src/Honeybee_Honeybee.py

```diff
@@ -244,6 +244,9 @@ def checkForUpdates(self, LB= True, HB= True, OpenStudio = True, template = True
                     with open(thermFile) as tempFile:
                         currentThermVersion = eval(tempFile.readline().split("!")[-1].strip())["version"]
                 except: isNewerThermAvailable = True
+
+                with open(thermFile) as tempFile:
```

Comment:
This line looks duplicated! we either need this or 244 to 246. I believe the one on the top is the one to be kept.
Am I right @chriswmackey?